tests/rt.conf

```diff
@@ -187,3 +187,10 @@ RUN     | datm_cdeps_debug_cfsr
 
 COMPILE | APP=ATMW SUITES=FV3_GFS_v16                                                                                           | - wcoss_cray  | fv3 |
 RUN     | control_atmwav                                                                                                        | - wcoss_cray  | fv3 |
+
+###################################################################################################################################################################################
+# ATM-GOCART tests                                                                                                                                                                   #
+###################################################################################################################################################################################
+
+COMPILE | APP=ATMAERO SUITES=FV3_GFS_v16                                                                                        | + hera.intel  | fv3 |
+RUN     | control_atm_aerosols                                                                                                  | + hera.intel  | fv3 |
```

Comment:
My understanding is that this test will run on all supported platforms. If so, please remove `+ hera.intel`

Reply:
Done in f1519bcd493ebd2adee3e0b7f94dda69bf77ff17